styles2.css

```diff
@@ -0,0 +1,209 @@
+:root {
+  --text-color1: #172b4d;
+  --text-green: #36b37f;
+}
+
+@font-face {
+  font-family: Inter;
+  src: url('./static/Inter-Light.ttf');
+}
+
+@font-face {
+  font-family: Inter;
+  font-weight: bold;
+  src: url('./static/Inter-Bold.ttf');
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+}
+
+span {
+  color: var(--text-green);
+}
+
+/* As we see in fring template, when we have a mobile device, the nav tag
+  is before the #main part, and it have space around
+*/
+
+#smallScreen {
+  display: flex;
```

Comment:
- [ ] As mentioned by the previous reviewer, please make sure you align your header correctly as per the original design, currently, you did not make any changes related to this

<img width="762" alt="1" src="https://user-images.githubusercontent.com/77843050/212210218-bc138e14-2ae4-4d61-a26f-35443b7fa35a.png">